SixTrack/sixtrack.s

```diff
@@ -18492,6 +18828,16 @@ c$$$         endif
          !Moved from daten()
          crabph4(ix)=el(ix)
          el(ix)=0d0
+!--Wire
```

Comment:
You want to always force ed/ek/el to be 0 for these elements in the tracking, regardless of what the input file says? It would probably be better to just check (if they should really always be 0 in fort.2), and spit out an error if the input is invalid.